### PR TITLE
Apply validation throughout the form

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/18883
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-106-20tdlj28.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-106-20tdlj28.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-106-20tdlj28.web.app

**Review by kialj876 — COMMENTED** [review]
Looking good! I just have one comment in addition to fixing up the tests

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-106-20tdlj28.web.app

File: btr-web/btr-common-components/components/bcros/inputs/Address/Index.vue [hunk 1]
```diff
@@ -104,9 +104,8 @@
 <script setup lang="ts">
 import { z } from 'zod'
 import { FormError } from '#ui/types'
-import { getAddressLine1Validator, getAddressCityValidator, getAddressRegionValidator, getAddressPostalCodeValidator }
-  from '../../../../../btr-main-app/utils/validators'
-
+import { getAddressCityValidator, getAddressLine1Validator, getAddressPostalCodeValidator, getAddressRegionValidator }
+  from '../../../../utils/validation/validators'
```
Comment (kialj876) [inline btr-web/btr-common-components/components/bcros/inputs/Address/Index.vue:108]: I think you can omit the import here by adding an index file in the validation folder. At the very least you can reference it with '~/utils/validation/validators'

**Review by kialj876 — APPROVED** [review]
One last minor thing, but this is good to go I think

File: btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts [hunk 2]
```diff
@@ -77,20 +77,6 @@ describe('pages -> Beneficial Owner Change', () => {
     cy.get('[data-cy=add-new-btn]').should('not.have.attr', 'disabled')
   })
 
-  it('has expected done button functionality with add individual', () => {
```
Comment (kialj876) [inline btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts:80]: how come you removed this test? Is it covered elsewhere?

File: btr-web/btr-main-app/cypress/e2e/pages/formValidation.cy.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import payFeesForBtrRegsigin from '../../fixtures/payFeeForBtrRegsigin.json'
+
+describe('pages -> Beneficial Owner Change', () => {
```
Comment (kialj876) [inline btr-web/btr-main-app/cypress/e2e/pages/formValidation.cy.ts:3]: should update this description too

File: btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts (hunk 2 above)
Comment (patrickpeinanw) [reply btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts:80]: In summaryTable.cy.ts and reviewConfirm.cy.ts, we fill the full form and add some individuals. So the happy path for DONE button has been covered.

File: btr-web/btr-common-components/components/bcros/inputs/Address/Index.vue (hunk 1 above)
Comment (patrickpeinanw) [reply btr-web/btr-common-components/components/bcros/inputs/Address/Index.vue:108]: There is an index file under the utils folder. I will just export these functions there.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-106-20tdlj28.web.app

**Pull request merged**